src/instr/instr_private.h

```diff
@@ -398,7 +398,8 @@ typedef enum{
   TRACING_COMPUTING,
   TRACING_SLEEPING,
   TRACING_SCAN,
-  TRACING_EXSCAN
+  TRACING_EXSCAN,
+  TRACING_CUSTOM
 } e_caller_type ;
 
 typedef struct s_instr_extra_data {
```

Comment:
This whole file and this data type doesn't exist anymore; I think this is now in instr_private.hpp organized as a class: TIData. (With renamed attributes, e.g. datatype1 -> send_type, ...)